README.md

```diff
@@ -11,7 +11,8 @@ kind create cluster --config cluster-config.yaml --wait 120s
 
 2. Run tests
 ```bash
-go test -count 1 -timeout 1h -race -v -run Single
+export CLUSTER_CIDR="172.18.1.128/25" # for monolith suite
```

Comment:
Why do we need this for single cluster testing?

Reply:
we need this env for monolith suite

Reply:
Should we set this env in monolith example instead?

/cc @glazychev-art

Reply:
We can't. For the same reason that we can't move the same envs for the interdomain.
This is specific to the kind loadbalancer.